_rules/95e043.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+success_criterion: 
+- 1.4.2 # Audio Control
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Applicability
+
+This rule applies to any HTML `audio` or `video` element that has a `duration` of more than 3 seconds, or is set to `loop`, and that is not `paused` or `muted`. 
+
+## Expectation
+
+For each test target, a mechanism to pause or stop the audio, or control the audio volume independently from the overall system volume level, is provided as one of the first five focusable elements in the [document](#https://www.w3.org/TR/dom/#concept-document).
```

Comment:
I'm wondering if we should add a few expectations here to check:

1. that the pause mechanism is visible
2. that the pause mechanism has a label that indicates it can be used to pause the audio

Reply:
@WilcoFiers I have re-written the expectation in two parts. Part 1 addresses the element, part 2 addresses the mechanism.